### ISSUE TITLE
react-onsenui: Calling PropTypes validators directly is not supported by the `prop-types` package

### ISSUE THREAD

**Issue by airamrguez** [OnsenUI/OnsenUI#2025]
__Environment__ 

```
[Core]
    onsenui 2.3.1

[Framework]
    react 15.5.4

[Framework binding]
    react-onsenui 15.5.4

[Additional libraries]
    prop-types 15.5.8

[Platform]
    all
[Browser]
    all
```

__Encountered problem__

I think react-onsenui is affected by this issue: https://github.com/facebook/prop-types/issues/38

![screen shot 2017-05-23 at 08 26 30](https://cloud.githubusercontent.com/assets/1159448/26342038/ba991c94-3f96-11e7-9949-59e0707981bb.png)

Updating prop-types package to the latest version should fix this.

__How to reproduce__

Just set up a simple project, import any component from `react-onsenui` an render it.

**Comment by asial-matagawa** [comment]
@airamrguez 
Thank you for information!
It seems that the current `react-onsenui` uses `prop-types@15.5.8` due to our `yarn.lock`:
https://github.com/OnsenUI/OnsenUI/blob/265a8c6cc61fec536006ce6c902b12ecdaee0da1/bindings/react/yarn.lock#L3839

We will update `prop-types` and release `react-onsenui@1.3.3`.

**Comment by airamrguez** [comment]
@asial-matagawa cool!

**Comment by asial-matagawa** [comment]
@airamrguez 
I just released `react-onsenui@1.3.3` with the fix. Thank you!
https://unpkg.com/react-onsenui@1.3.3/

Closing this.
